platform-metrics/src/main/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistry.java

```diff
@@ -316,10 +316,10 @@ public static synchronized void stop() {
 
     DEFAULT_TAGS.clear();
     METER_REGISTRY.forEachMeter(METER_REGISTRY::remove);
-    METER_REGISTRY.getRegistries().forEach(e -> {
-      e.clear();
-      METER_REGISTRY.remove(e);
-    });
+    METER_REGISTRY.getRegistries().forEach(MeterRegistry::clear);
```

Comment:
As discussed offline, Should we call a `close` too before clearing?

Reply:
@buchi-busireddy  you might have a bit better understanding of . We noticed `close()` methods below.  Do we need to call in the stop block  ?

```registries.forEach(MeterRegistry::close);
    METER_REGISTRY.close();
```

Reply:
So, if we're not really stopping and re-initializing the `PlatformMetricRegistry` in a JVM, the right thing to do is just call `METER_REGISTRY.close()` and that will take care of closing all registered meter registries. 
However, I think we reuse the `METER_REGISTRY` static instance (in unit tests), hence this method was implemented accordingly. Actually, we should feel free to fix that as well.